src/main/kotlin/app/revanced/patches/youtube/layout/autorepeat/fingerprints/AutoRepeatFingerprint.kt

```diff
@@ -0,0 +1,43 @@
+package app.revanced.patches.youtube.layout.autorepeat.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patcher.fingerprint.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patches.youtube.layout.autorepeat.annotations.AutoRepeatCompatibility
+import org.jf.dexlib2.AccessFlags
+
+@Name("auto-repeat-fingerprint")
+@MatchingMethod(
+    "Ljvy;", "<init>"
+)
+@FuzzyPatternScanMethod(2) // FIXME: Test this threshold and find the best value.
+@AutoRepeatCompatibility
+@Version("0.0.1")
+//ToDo: Find method:
+/*
+public final void ae() {
+        aq(aabj.ENDED);
+    }
+ */
+//Change method to:
+/*
+public final void ae() {
+    aq(aabj.ENDED);
+    app.revanced.integrations.sponsorblock.player.VideoInformation.videoEnded();
```

Comment:
@oSumAtrIX the SB Patch needs this method to be called. How do we do that? Should we leave it for this patch rn and then figure a way out later?